src/locales/en_US.ts

```diff
@@ -1931,6 +1933,20 @@ const strings = {
   see_all: 'See All',
   sell_crypto: 'Sell Crypto',
   sell_crypto_footer: 'Crypto to bank or cash',
+
+  // Trade Option Select Scene
+  trade_option_buy_title: 'Buy Cryptocurrency',
```

Comment:
Do not prefix string with where they are used. This makes it hard to utilize identical strings in different places. Instead prefix them with what type of UI element they are used for. ie

`choose_provider_button`
`total_label`
`powered_by_button`

For longer text, you can just use the first few words of the text

 `please_try_again_no_providers`

Reply:
Rename this lstring to `buy_cryptocurrency_scene_title`.